From 7c5eed62a6474409f5859b635d41df6c97b8446f Mon Sep 17 00:00:00 2001
From: Guido Stet <info@guidostet.com>
Date: Fri, 29 Nov 2024 00:20:11 +0100
Subject: [PATCH] Fix implicit type specifier errors

explicit int type specifier to resolve:
80:1: error: type specifier missing, defaults to 'int'
93:1: error: type specifier missing, defaults to 'int';
ISO C99 and later do not support implicit int [-Wimplicit-int]
---
 util/rarian-example.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/util/rarian-example.c b/util/rarian-example.c
index d203a19..87fbcf2 100644
--- util/rarian-example.c
+++ util/rarian-example.c
@@ -77,6 +77,7 @@ for_each_cb (RrnReg *reg, void * user_data)
     return TRUE;
 }
 
+int
 info_for_each (RrnInfoEntry *entry, void *data)
 {
   if (entry->section)
@@ -90,6 +91,7 @@ info_for_each (RrnInfoEntry *entry, void *data)
   return TRUE;
 }
 
+int
 man_for_each (RrnManEntry *entry, void *data)
 {
   printf ("Man page %s\n", entry->name);
